src/App.vue

```diff
@@ -43,7 +43,12 @@
             class="beer-list__empty"
           >No beers available to show, please try adjusting your filter criteria, or adding your favourite beer.</p>
 
-          <Beer v-for="beer in filterBeers" :beer="beer" :key="beer.id" />
+          <Beer
+            v-for="beer in filterBeers"
+            :beer="beer"
+            @sendFavouriteId="updateMyFavourites"
```

Comment:
It is usual for event to be kebab case to respect html like attributes. The vue eslint rules does ensure it is kebab case and you might see it in the future or in the doc. 

This is more for information so you're not surprised if you see it kebab case.

Reply:
I have changed the emit event to kebab case.

```this.$emit('send-favourite-id', this.beer.id);```

Does the v-on need to change? I read this  >https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

And I think it's ok to leave, but please can you confirm its ok. Thanks

Reply:
Yes it needs to match 👍 
```@send-favourite-id="updateMyFavourites"```